src/crossroads/main.py

```diff
@@ -88,24 +88,21 @@ def __init__(self, location):
         self.longitude = location["lng"]
         self.infobox = location["infobox"]
 
-def backend():
-    df_gps = pd.read_json('sample.json')
-    print('There are {:,} rows in the location history dataset'.format(len(df_gps)))
+def json_parser():
+    """
+    Parse .json file with Location History from Google Maps timeline.
 
-    # parse lat, lon, and timestamp from the dict inside the locations column
-    df_gps['lat'] = df_gps['locations'].map(lambda x: x['latitudeE7'])
-    df_gps['lon'] = df_gps['locations'].map(lambda x: x['longitudeE7'])
-    df_gps['timestamp_ms'] = df_gps['locations'].map(lambda x: x['timestampMs'])
+    :param jfile: JSON file
+    :return: a list of chronologically arranged dictionaries, from the oldest ones
+    """
+    #test version with "sample.json" - swap TODO
+    x = open("sample.json", "r")
```

Comment:
`x` variable is not meaningful for code reader. It's better to use more explicit names like `json_coordinates_file`.
Additionally it's better to use context manager `with` for operation on files. You already forget to close this file and context manager will do that for you.